extensions/fonts/hooks-configure-fonts

```diff
@@ -40,6 +40,7 @@ EOF
 
 export FONTCONFIG_FILE="${SNAP_COMMON}/fontconfig/fonts.conf"
 
-"${SNAP_DESKTOP_RUNTIME}/usr/bin/fc-cache" --force --system-only --verbose || echo "configure hook: font generation failed"
-
-exec "$@"
+if [ -f "${SNAP_DESKTOP_RUNTIME}/usr/bin/fc-cache" ]; then
```

Comment:
I think this is fine, but I think commenting out the exec wasn't intended?

Reply:
the only problem I see with this is that if the runtime is not connected, this will silently go by and we will have a hard time debugging why something is slow on a given system

Reply:
I added an ELSE for that. I'll upload it now.

Reply:
Uploaded.

Reply:
About commenting the exec: again, it failed. But I have to check if now that gnome-42 is plugged automagically, that error still persists.

Reply:
![imagen](https://user-images.githubusercontent.com/105285003/171378705-7c6d7a36-1687-4abd-ae49-97c207d31b68.png)

I receive that error during installation if I leave the final "exec"

Reply:
Ok, I added a test at the exec too.